src/main/java/racing/controller/Game.java

```diff
@@ -0,0 +1,41 @@
+package racing.controller;
+
+import racing.domain.Cars;
+import racing.domain.MoveStrategy;
+import racing.domain.RaceResult;
+import racing.domain.RandomMoveStrategy;
+import racing.domain.RandomNumberGenerator;
+import racing.domain.RoundResult;
+import racing.view.OutputView;
+
+public class Game {
+
+    private final Cars cars;
+    private final int tryCount;
+
+    public Game(int carCount, int tryCount) {
+        this.cars = new Cars(carCount);
+        this.tryCount = tryCount;
+    }
+
+    public void race() {
+        MoveStrategy moveStrategy = createMoveStrategy();
+        RaceResult raceResult = new RaceResult();
+
+        for (int i = 0; i < tryCount; i++) {
+            playSingleRound(moveStrategy, raceResult);
+        }
+
+        OutputView.printRaceResult(raceResult);
+    }
+
+    private MoveStrategy createMoveStrategy() {
+        return new RandomMoveStrategy(new RandomNumberGenerator());
+    }
+
+    private void playSingleRound(MoveStrategy moveStrategy, RaceResult raceResult) {
+        cars.moveCars(moveStrategy);
+        RoundResult roundResult = new RoundResult(cars);
```

Comment:
```suggestion
        RoundResult roundResult = cars.toRoundResult();
```
Cars가 racing한 모든 결과 값을 가지고 있어 위와 같이 구현하는 것은 어떨까?